linkml/generators/pythongen.py

```diff
@@ -127,9 +126,11 @@ def gen_schema(self) -> str:
         handlerimport = (
             "from linkml_runtime.utils.enumerations import EnumDefinitionImpl"
         )
-        split_descripton = "\n#              ".join(
-            split_line(be(self.schema.description), split_len=100)
-        )
+        split_description = ""
+        if self.schema.description:
+            split_description = "\n#   ".join(
+                d for d in self.schema.description.split("\n") if d is not None
+            )
```

Comment:
any formatting in the description was getting lost so this change ensures formatting is retained